contracts/ComptrollerStorage.sol

```diff
@@ -137,13 +137,18 @@ contract ComptrollerV2Storage is ComptrollerV1Storage {
     /// @notice Indexes of account addresses in the `whitelistArray` array
     mapping(address => uint256) internal whitelistIndexes;
 
+    /// @notice Maps CTokens to Max Supply cap indicating what should be the maximum supply cap for all assets
```

Comment:
This comment is misleading. It maps each cToken to **it's own supply cap**, not "the supply cap for all assets"

Reply:
Yeah! I agree I will make changes

Reply:
great!

Reply:
made Changes @TransmissionsDev